SPECS/python3/pathfix.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python3
```

Comment:
Is this an original microsoft script? Or did it come from somewhere else?

Reply:
No, this was previously provided in the python 3.9 source, and was removed from the 3.12 source. Link to original: https://github.com/python/cpython/blob/3.9/Tools/scripts/pathfix.py